extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -103,7 +106,24 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', [
 				width = newWidth;
 				height = newHeight;
 
-				ima.resize(width, height);
+				if (params.isFullscreen) {
+					ima.resize(ima.viewMode.FULLSCREEN);
+				} else {
+					ima.resize(ima.viewMode.NORMAL, width, height);
+				}
+			},
+			toggleFullscreen: function () {
+				if (params.isFullscreen) {
+					ima.resize(ima.viewMode.NORMAL, width, height);
+					this.container.classList.add(videoFullscreenClassName);
```

Comment:
if we're doing here direct resize `ima.resize` instead of calling `this.resize()` the height & width props of this object won't be updates. Is this on purpose?

Reply:
yes, on purpose. I want to still update with/height in the background during resizing, so when you will enter to windowed mode again, it will hava s proper size.